postgresql.conf.jinja

```diff
@@ -316,6 +316,45 @@ log_min_duration_statement = '1500ms'
 log_line_prefix = '%m [%p] %q%u@%d '
 log_timezone = 'Europe/Berlin'
 
+# Controls which SQL statements are logged. Valid values are none (off), ddl, mod, and all (all statements).
+# ddl logs all data definition statements, such as CREATE, ALTER, and DROP statements. mod logs all ddl
+# statements, plus data-modifying statements such as INSERT, UPDATE, DELETE, TRUNCATE, and COPY FROM. PREPARE, 
+# EXECUTE, and EXPLAIN ANALYZE statements are also logged if their contained command is of an appropriate type. 
+{%- if log_statement is defined %}
+log_statement = '{{ log_statement }}'
+{%- else %}
```

Comment:
Skip the else branch - then the current postgresql default is used (this is a little more stable in case the default changes in the next postgresql release)